<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>39</y>
      <width>991</width>
      <height>621</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="login">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>160</y>
        <width>271</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>SignIn</string>
      </property>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>user Name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="user">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="pass">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_login">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>221</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="mainmenu">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1071</width>
        <height>491</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Ajouter un employee</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>50</y>
          <width>471</width>
          <height>361</height>
         </rect>
        </property>
        <property name="title">
         <string>Ajout</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>60</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>110</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Prenom</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>160</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Cin</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_nom">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>60</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_prenom">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>110</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_cin">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>160</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>210</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Identifiant</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>260</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Tache</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_id">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>210</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_tache">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>260</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_ajouter">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>140</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_age">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>310</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>310</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>age</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Afficher les employee</string>
       </attribute>
       <widget class="QTableView" name="tab_employee">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>591</width>
          <height>281</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>30</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>   Nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>70</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>   Prenom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>120</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>    Cin</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>210</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string> Identifiant</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>160</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string> Tache</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="nv_nom">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>30</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="nv_prenom">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>70</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="nv_tache">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>160</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="nv_cin">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>120</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="nv_id">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>210</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_modifier">
        <property name="geometry">
         <rect>
          <x>910</x>
          <y>110</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_supprimer">
        <property name="geometry">
         <rect>
          <x>910</x>
          <y>180</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>supprimer</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="cin_rech">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>330</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_rech">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>330</y>
          <width>111</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>rechercher par cin</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_tri">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>390</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>trier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_stat">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>390</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>statistique</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>260</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>age</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="nv_age">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>260</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>431</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>par nom</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>par prenom</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>par age</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>chat </string>
       </attribute>
       <widget class="QPushButton" name="pb_server">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>141</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>lancer le serveur</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>330</y>
          <width>51</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Message</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_envoyer">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>330</y>
          <width>61</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Envoyer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_connecter">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>330</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Connecter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>61</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Nickname</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_nickname">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>70</y>
          <width>411</width>
          <height>19</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_message">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>330</y>
          <width>291</width>
          <height>19</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>471</width>
          <height>222</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>330</y>
          <width>51</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Message</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_envoyer_2">
        <property name="geometry">
         <rect>
          <x>850</x>
          <y>330</y>
          <width>61</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Envoyer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_connecter_2">
        <property name="geometry">
         <rect>
          <x>910</x>
          <y>330</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Connecter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>70</y>
          <width>61</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Nickname</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_nickname_2">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>70</y>
          <width>411</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_message_2">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>330</y>
          <width>301</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_2">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>100</y>
          <width>471</width>
          <height>222</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1122</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
